doc/pages/administrator/030-installation_kubernetes.md

```diff
@@ -86,17 +86,67 @@ kubectl delete deployments.app -n kube-system astarte-operator
 **DO NOT** delete Astarte's CRDs! This will lead to the deletion of the entire Astarte deployment
 with a consequent data loss.
 
-To restore the Operator's functionalities with v1.0, all you have to do is following the install
-instructions, i.e.:
+Please, before proceeding to the next steps make sure the requirements listed in the
```

Comment:
Not sure I'd use this wording. The only change in requirement is cert-manager, so I'd rather point out that cert-manager should be installed in the cluster if it isn't, with the correct setup